CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -1759,11 +1795,26 @@ private void OnWindowStateChanged(object sender, EventArgs e)
         /// even when it's not displayed on screen.
         /// </summary>
         /// <param name="hidden">if true the browser will be notified that it was hidden.</param>
-        protected virtual void OnBrowserWasHidden(bool hidden)
+        protected virtual async void OnBrowserWasHidden(bool hidden)
         {
             if (browser != null)
             {
                 browser.GetHost().WasHidden(hidden);
+
+                if (hidden)
+                {
+                    if (EnableResizeHackForIssue2779)
+                    {
+                        resizeHackForIssue2779Enabled = true;
+                    }
+                }
+                else
+                {
+                    await CefUiThreadRunAsync(async () =>
```

Comment:
Let's remove the async modifier from the method and discard the Task returned from the method. 

There's no continuation.